### PR TITLE
371 seed data should be inserted into the database

### PR DESCRIPTION
**Pull request by Ayako-Yokoe**
## Proposed changes

- OpenTree Education is used as a general principal. So, principal_id is set to @principalId in the competencies 
- Five categories are Functional, Strategic, Organizational, Behavioural, Operational and each category has an image-URL
- Competencies table has 'label', 'description', 'principal_id', 'category_id'

## Screenshots

Migration build on flyway:
Screen Shot 2022-06-27 at 13 15 04

Updated Database:
Screen Shot 2022-06-27 at 13 15 41

Screen Shot 2022-06-27 at 13 16 16

## Checklist

- [x] Are the issues being addressed linked to this PR?
- [x] Do all commit messages start with the issue number?
- [x] Are all code changes sufficiently tested?
- [x] Are there screenshots for UI changes?

### CODE REVIEW

**Review by achichikalova — APPROVED** [review]
Amazing job, Ayako! Well done!

**Comment by tenzinlekphell** [comment]
Looks good Ayako!

**Review by tenzinlekphell — APPROVED** [review]

File: db/migration/V017__add_categories_and_competencies_data.sql
Comment (Nazanin-Rezaei) [inline db/migration/V017__add_categories_and_competencies_data.sql:11]: Great, we have all images for all categories in our database.

Comment (Ayako-Yokoe) [reply db/migration/V017__add_categories_and_competencies_data.sql:11]: It seems I need a code owner review. Once it's approved, I'll do a merge.

File: db/migration/V017__add_categories_and_competencies_data.sql
Comment (daveytech) [inline db/migration/V017__add_categories_and_competencies_data.sql:2]: Let's keep the whitespace minimal by removing the extra empty lines in between.

File: db/migration/V017__add_categories_and_competencies_data.sql
Comment (daveytech) [inline db/migration/V017__add_categories_and_competencies_data.sql:4]: Remove whitespace lines.

File: db/migration/V017__add_categories_and_competencies_data.sql
Comment (daveytech) [inline db/migration/V017__add_categories_and_competencies_data.sql:25]: Let's remove these extra whitespace lines. If you prefer some organizational separation of the categories then 1 line is sufficient.

**Review by daveytech — CHANGES_REQUESTED** [review]
We can clean up the file just a tad by removing the extra white space lines. You are welcome to keep one line between the competencies by category to keep some organization.

**Review by daveytech — APPROVED** [review]
Thanks for the changes!

**Pull request merged**
